jmxtrans-output/jmxtrans-output-logback/pom.xml

```diff
@@ -68,6 +68,10 @@
 			<groupId>org.jmxtrans</groupId>
 			<artifactId>jmxtrans-utils</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
pom.xml should be sorted according to standard. You can use `./mvnw sortpom:sort` to sort it automatically.

Reply:
Thanks for the reviews! I will correct all these mistakes.